roles/sap_control/tasks/prepare.yml

```diff
@@ -13,9 +13,9 @@
   vars:
     sap_control_execute_sid: "{{ item.SID }}"
     sap_control_execute_type: "{{ item.Type }}"
-    sap_control_execute_instance_nr: "{{ item.InstanceNumber }}"
+    sap_control_execute_instance_nr: "{{ item.NR }}"
```

Comment:
```suggestion
    sap_control_execute_instance_nr: "{{ item.InstanceNumber }}"
```

Reply:
Hi Gabriele,
after double checking all branches of sap_libs, I only see the variable "NR" for the InstanceNumber in the result from module sap_libs.sap_system_facts.

Here the result from sap_system_facts in a play:
![image](https://github.com/user-attachments/assets/ede38567-ee08-465f-aa0f-3c78888e2c40)

Do I miss something?

Regards,
Rainer

Reply:
I was wrong, I remembered wrong:
https://github.com/sap-linuxlab/community.sap_libs/blob/7d8bc0f11c2e1f58da73710e7925f456d011ec0c/plugins/modules/sap_system_facts.py#L112